runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/resources/ClasspathScanningResourcesDetector.java

```diff
@@ -45,7 +45,11 @@ public ClasspathScanningResourcesDetector(ClassGraph classGraph) {
   @Override
   public List<String> detect(ClassLoader classLoader) {
     List<File> classpathContents =
-        classGraph.disableNestedJarScanning().addClassLoader(classLoader).getClasspathFiles();
+        classGraph
+            .disableNestedJarScanning()
+            .addClassLoader(classLoader)
+            .scan(1)
```

Comment:
Could you add a quick comment about this workaround?

Reply:
@steveniemitz Please follow up with a separate PR if this is not addressed.